src/imagery/i.eodag/i.eodag.py

```diff
@@ -255,37 +330,62 @@ def setup_environment_variables(env, **kwargs):
         if extract:
             gs.warning(
                 _(
-                    """Ignoring 'e' flag...
-                    'extract' option in the config file will be used.
-                    If you wish to use the 'e' flag, please specify a provider."""
+                    "Ignoring 'e' flag... \
+                    'extract' option in the config file will be used. \
```

Comment:
Are these `\` supposed to be new lines? Shouldn't that be `\n`?

Reply:
I was leaving the part of print new lines to `gs.warning`, and these backslashes `\` was to be able to write a string on multiple lines (just in the code). 
But yes you are right, I added `\n` as well now.